docs/recipes/setup_circleci.md

```diff
@@ -0,0 +1,30 @@
+# How to: setup CircleCI to deploy to cloud.gov
+
+The cloud.gov docs recommend using a [space-deployer](https://cloud.gov/docs/services/cloud-gov-service-account/) to deploy applications from continuous integration services such as CircleCI.
+
+It is worth noting that you will need to have at least `SpaceManager` access to the cloud.gov space for which you are trying to create a `SpaceDeployer` service.
+
+It is also worth noting that you will need admin access to the HHS OPRE-OPS CircleCI project in order to configure the cloud.gov environment variables inside CircleCI. So, first, make sure you have all the correct permissions required and then proceed with the steps below.
+
+```
+# first, log in:
+cf login --sso
+
+# then, create a service named "circleci-deployer-account" by running this command:
+cf create-service cloud-gov-service-account space-deployer circleci-deployer-account
+
+# then, create a service key for the newly-create service account by running this command:
+cf create-service-key circleci-deployer-account circleci-deployer-key               
+
+# then, retrieve the created service key for the "circleci-deployer-account" by running this command:
+cf service-key circleci-deployer-account circleci-deployer-key
+```
+
+The last command will return a username/password like this:
+```
+{
+ "password": "oYasdfliaweinasfdliecV",
+ "username": "deadbeed-aabb-1234-feha0987654321000"
+}
+```
+These are the credentials you will need to insert into the CircleCI OPRE-OPS project environment variables the names of which will need to correspond to the environment variables in the deploy scripts in `.circleci/config.yml`
```

Comment:
[nitpick] Maybe adding something like:
>i.e. $DEV_USER and $DEV_PASSWORD for our case

makes it a bit more explicit what you are referring to.

Reply:
I went back and forth on this because they'll need to do this for not just $DEV_ but it might be better to be more explicit.

Reply:
I was thinking it may not be apparent what we are referring to.  If we don't want to use `$DEV_USER`, may be specified that this is the credentials you need to supply when logging into cloud.gov

Reply:
Between the two, I think I'll just use `$DEV_USER` for now, because these generated creds are not something they should use when logging into cloud.gov